tests/components/participants/helpers/participantHelper.test.ts

```diff
@@ -0,0 +1,24 @@
+import '@testing-library/jest-dom';
+import { applyFilters } from '../../../../src/components/participants/helpers/participantHelper';
+
+import { mockParticipants } from './__fixtures__/participants';
+
+describe('ParticipantHelper_ApplyFilters', () => {
+  it('filters correctly the participant list', () => {
+    const searchText = 'msg systems ag';
+
+    // Apply filters based on the search text
+    const filteredParticipants = applyFilters(mockParticipants, searchText);
+
+    // Check if two participants match the search text
+    expect(filteredParticipants).toHaveLength(2);
+
+    // Check that the participant is the expected one (mockParticipants[0])
+    // Assuming "msg systems AG" is mockParticipants[0]
+    expect(filteredParticipants).toContain(mockParticipants[0]);
```

Comment:
Please do not use indexed element as expected element. See similar comments above.